src/App.css

```diff
@@ -1,3 +1,49 @@
+.container {
+  max-width: 100%;
+  display: flex;
+  flex-direction: row;
+  padding-right: 0px;
+  padding-left: 0px;
+  margin-top: 10px;
+  margin-left: 0px;
+  margin-right: 0px;
+}
+.flexItem {
+  flex-direction: row;
+  display: flex;
+  flex-wrap: nowrap;
+  color: #666;
+  justify-content: flex-start;
+  align-items: flex-start;
+  margin-bottom: 20px;
+}
+
+label {
+  display: inline;
+}
+input[type="checkbox"] {
+  margin-top: 10px;
+  bottom: 1px;
+  width: 50px;
+  -webkit-appearance: checkbox;
+  outline: none;
+}
+
+.dg_service_counter {
+  text-align: right;
+  font-style: italic;
+  color: gray;
+}
+
+.dg_checkbox {
```

Comment:
i don't think this should be here, i suspect soemthing is up.

Reply:
removed